packages/@aws-cdk/aws-events-targets/lib/log-group-resource-policy.ts

```diff
@@ -0,0 +1,50 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as cdk from '@aws-cdk/core';
+import * as cr from '@aws-cdk/custom-resources';
+
+/**
+ * Construction properties for LogGroupResourcePolicy
+ */
+export interface LogGroupResourcePolicyProps {
+  /**
+   * The log group resource policy name
+   */
+  readonly policyName: string;
+  /**
+   * The policy statements for the log group resource logs
+   */
+  readonly policyStatements: [iam.PolicyStatement];
+}
+
+/**
+ * Creates LogGroup resource policies.
+ */
+export class LogGroupResourcePolicy extends cr.AwsCustomResource {
+  constructor(scope: cdk.Construct, id: string, props: LogGroupResourcePolicyProps) {
+    const policyDocument = new iam.PolicyDocument({
+      statements: props.policyStatements,
+    });
+
+    super(scope, id, {
+      resourceType: 'Custom::CloudwatchLogResourcePolicy',
+      onUpdate: {
+        service: 'CloudWatchLogs',
+        action: 'putResourcePolicy',
+        parameters: {
+          policyName: props.policyName,
+          policyDocument: JSON.stringify(policyDocument),
+        },
+        physicalResourceId: cr.PhysicalResourceId.of(id),
+      },
+      onDelete: {
+        service: 'CloudWatchLogs',
+        action: 'deleteResourcePolicy',
+        parameters: {
+          policyName: props.policyName,
+        },
+        ignoreErrorCodesMatching: '400',
+      },
+      policy: cr.AwsCustomResourcePolicy.fromSdkCalls({ resources: ['*'] }),
```

Comment:
nit: been asking in PRs recently to add a comment any time we're using `*` to better comment justification in the code. do you mind adding the rationale in?